scammer-label-propagation/src/agent.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import os
+import time
+import forta_agent
+from web3 import Web3
+from forta_agent import get_json_rpc_url, Finding
+from concurrent.futures import ThreadPoolExecutor
+
+from src.main import run_all
+from src.constants import attacker_bots, ATTACKER_CONFIDENCE, N_WORKERS, MAX_FINDINGS
+
+
+# If we are in production, we log to the console. Otherwise, we log to a file
+if 'NODE_ENV' in os.environ and 'production' in os.environ.get('NODE_ENV'):
```

Comment:
you can just do this
```suggestion
if 'production' in os.environ.get('NODE_ENV', ''):
```

Reply:
corrected